arduino-ide-extension/src/electron-browser/electron-app-service.ts

```diff
@@ -0,0 +1,26 @@
+import type { Disposable } from '@theia/core/lib/common/disposable';
+import type { MaybePromise } from '@theia/core/lib/common/types';
+import { injectable } from '@theia/core/shared/inversify';
+import type { AppService } from '../browser/app-service';
+import type { Sketch } from '../common/protocol/sketches-service';
+
+@injectable()
+export class ElectronAppService implements AppService {
+  quit(): void {
+    window.electronArduino.quitApp();
+  }
+
+  version(): Promise<string> {
+    return window.electronArduino.appVersion();
+  }
+
+  registerStartupTasksHandler(
+    handler: (args: unknown) => MaybePromise<void>
```

Comment:
can be `handler: (tasks: StartupTasks)` from electron api.
should be => void

Reply:
https://github.com/arduino/arduino-ide/pull/2027/commits/241baca40be0ed10b4d872c65f96d9034190f358